contracts/football_bets.py

```
@@ -40,37 +48,40 @@ def get_match_result() -> str:
+Only return valid JSON without any extra text.
+"""
+
+            result = gl.nondet.exec_prompt(
+                task,
+                response_format="json"
+            )
             return json.dumps(result, sort_keys=True)
 
-        result_json = json.loads(gl.eq_principle.strict_eq(get_match_result))
+        result_json = json.loads(
+            gl.eq_principle.strict_eq(get_match_result)
```

Ruff (0.15.7)

[error] `gl` may be undefined, or defined from star imports

(F405)